core/message_formatter.py

```
@@ -6,194 +6,200 @@
-    paragraphs = text.split('\n\n')
-    
+    paragraphs = text.split("\n\n")
+
     formatted_paragraphs = []
     for paragraph in paragraphs:
         paragraph = paragraph.strip()
         if paragraph:
             # Ensure sentences end with proper punctuation
-            if paragraph and not paragraph[-1] in '.!?':
-                paragraph += '.'
+            if paragraph and not paragraph[-1] in ".!?":
```

Ruff (0.13.1)

Test for membership should be `not in`

Convert to `not in`

(E713)